sklearn/utils/fixes.py

```diff
@@ -393,3 +393,15 @@ def array_equal(a1, a2):
         return bool(np.asarray(a1 == a2).all())
 else:
     from numpy import array_equal
+
+
+def safe_median(arr, *args, **kwargs):
```

Comment:
I'm not sure we need this here as it is only used in the test. At least we should make it private.